class_tree.gd

```diff
@@ -44,73 +44,112 @@ func generate_full_node_list() -> void:
 	# Generate the full list of node classes once
 	full_node_list.clear()  # Ensure the list is empty before populating
 	var node_classes = ClassDB.get_inheriters_from_class("Node")
+	node_classes.append("Node")  # Manually add "Node"
+
+	print("Total inheriters from Node (including 'Node'):", node_classes.size())
```

Comment:
you should consider wrapping these debug prints under certain condition

Reply:
man these prints are killing me , chat gpt keeps adding them , i was going to delete all of them but then i said no cuz they might be needed by you or someone else for debugging purposes , feel free to delete any of them if u want